src/index.html

```diff
@@ -11,7 +11,142 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form 
+    action="http://localhost:8080/" 
+    method="post" 
+    autocomplete="off">
+      <fieldset class="form-field">
+        <legend>Personal information:</legend> 
+        <label class="form-field">
+          Surname:
+          <input 
+            type="text" 
+            name="Surname:" 
+            required
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input 
+            type="text" 
+            name="Name" 
+            required
+          >
+        </label>
+        <label class="form-field">
+          How old are you?
+          <input type="number" 
+            name="How old are you?" 
+            value="12" 
+            min="1" 
+            max="100"
+          >
+        </label>
+        <label class="form-field">
+        Full date of birth:
+          <input 
+            type="date" 
+            name="Full date of birth:" 
+            required 
+            placeholder="mm/dd/yyyy"
+          >
+        </label>
+        <label class="form-field">
+          <input 
+            type="checkbox"
+          >
+            I accept the term of the agreement
+        </label>
+      </fieldset>
+      <fieldset class="form-field">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input 
+            type="email" 
+            name="E-mail" 
+            required 
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input 
+            type="password " 
+            name="Password" 
+            required
+            minlength="6"
+            maxlength="12"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-field">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">
+        Do you love cats?
+          <input 
+            type="radio" 
+            name="answer" 
+           value="yes"
+          >
+        Yes
+          <input 
+            type="radio" 
+            name="answer" 
+            value="no"
+          >
+        No
```

Comment:
let's cover each input in the label tag to avoid this strange behavior

https://user-images.githubusercontent.com/70220667/229272534-1f21d2ca-ef6c-4946-a8e1-c134e0ecdd63.mov